app/src/main/java/dev/arkbuilders/arkmemo/graphics/SVG.kt

```diff
@@ -159,6 +159,8 @@ private const val MOVE_TO = "M"
 private const val ABS_LINE_TO = "L"
 private const val ABS_QUAD_TO = "Q"
 
+private const val ZERO = "0"
+private const val HUNDRED = "100"
```

Comment:
Name of the constant isn't helpful. This is a _dimension_ of the box, right?

Reply:
Yes :)